app/config.js

```diff
@@ -152,7 +152,7 @@ const config = {
     documentUpload: {
         validMimeTypes: ['image/jpeg', 'image/bmp', 'image/tiff', 'image/png', 'application/pdf'],
         maxFiles: 10,
-        maxFileSizeMb: 10
+        maxSizeBytes: 10000000
```

Comment:
Not sure how picky you want to be about it but I think this is set up in the backend to be -> 10485760 bytes.